addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/AccountHandler.java

```diff
@@ -390,27 +393,51 @@ private void refreshData() {
                 updateDeviceList();
                 updateFlashBriefingHandlers();
 
-                // update notification states
                 DeviceNotificationState[] deviceNotificationStates = null;
+                AscendingAlarmModel[] ascendingAlarmModels = null;
+                JsonBluetoothStates states = null;
+                List<JsonMusicProvider> musicProviders = null;
                 if (currentConnection.getIsLoggedIn()) {
+
```

Comment:
Sorry for being nit-picking: May I ask you to remove this empty line?

Reply:
No problem